_episodes/includes.md

```diff
@@ -149,7 +149,7 @@ at the top of your page.
 > > ## Solution
 > >
 > > ```
-> > <a href="https://YOUR_USERNAME.github.io/YOUR_REPO_NAME/"><img src="/images/site_banner.png" alt="Group Website with Jekyll"></a>
+> > <a href="https://YOUR_USERNAME.github.io/YOUR_REPO_NAME/"><img src="images/site_banner.png" alt="Group Website with Jekyll"></a>
```

Comment:
I can see this break if the include done later is done from another folder. Either way, neither prefixing with / nor leaving it out is going to work for all cases.

Reply:
Wondering if it makes sense to turn this into an exercise in "Page layouts"...